tough/src/cache.rs

```diff
@@ -136,14 +145,7 @@ impl<'a, T: Transport> Repository<'a, T> {
     /// Saves a signed target to the specified `outdir`. Retains the digest-prepended filename if
     /// consistent snapshots are used.
     fn cache_target<P: AsRef<Path>>(&self, outdir: P, name: &str) -> Result<()> {
-        let t = self
-            .targets
-            .signed
-            .targets
-            .get(name)
-            .context(error::CacheTargetMissing {
-                target_name: name.to_owned(),
-            })?;
+        let t = self.targets.signed.find_target(name).unwrap();
```

Comment:
Need to retain the context(error::CacheTargetMissing), we don't use `unwrap()` in production code (test code is fine).

```suggestion
        let t = self.targets.signed.find_target(name)..context(error::CacheTargetMissing {
                target_name: name.to_owned(),
            })?;
```